spec/ddtrace/configuration/settings_spec.rb

```diff
@@ -77,8 +77,8 @@
         expect(Datadog::Logger.log).to eq(custom_log)
         expect(tracer.writer.transport.current_api.adapter.hostname).to eq('tracer.host.com')
         expect(tracer.writer.transport.current_api.adapter.port).to eq(1234)
-        expect(tracer.tags[:env]).to eq(:config_test)
-        expect(tracer.tags[:foo]).to eq(:bar)
+        expect(tracer.tags['env']).to eq(:config_test)
```

Comment:
I think changing this behavior is okay?

Reply:
hmm... I guess it depends if people are accessing tags directly via symbols?

otherwise, this shouldn't break anything on encoding or sending to the agent